src/app/components/sandbox-controller/sandbox-controller.component.html

```diff
@@ -11,26 +11,22 @@
   </div>
 </div>
 
-<ng-container *ngIf="canAddSource$ | async; else hrule">
-  <div class="row">
-    <div class="col d-flex justify-content-center">
-      <button
-        mat-icon-button
-        matTooltip="Add input stream"
-        aria-label="Add input stream"
-        (click)="addInputStream()"
-      >
-        <mat-icon>add_box</mat-icon>
-      </button>
-    </div>
+<div class="row">
+  <div class="col d-flex justify-content-center">
+    <button
+      *ngIf="canAddSource$ | async"
+      mat-icon-button
+      matTooltip="Add input stream"
```

Comment:
It feels like the tooltips are just getting in the way and making the UI too busy without adding much value.
What do you think? Remove them for the add stream, share, and visualize buttons?

Reply:
I can dig it, will keep the `aria-label` for screen readers 👍🏼